continuous_integration/environment-3.6.yaml

```
@@ -57,3 +57,5 @@ dependencies:
   - python-xxhash
   - mmh3
   - pip
+  - pip:
+    - git+https://github.com/dask/distributed
```

We may need [this workaround borrowed from `distributed`]( https://github.com/dask/distributed/blob/7f8bb81701c747f042dc8e5d19c5c7fec09de67a/continuous_integration/environment-3.6.yaml#L42-L46 )

```suggestion
    - git+https://github.com/dask/distributed.git@master
```

---

Good thinking. I was wondering about that myself, but it looks like the correct version of `distributed` of getting installed (see https://github.com/dask/dask/pull/7279/checks?check_run_id=1989339554#step:5:82) in the Python 3.6 CI builds. This is what I get when using the latest `distributed` commit (https://github.com/dask/distributed/commit/7f8bb81701c747f042dc8e5d19c5c7fec09de67a) locally:

```python
In [1]: import distributed

In [2]: distributed.__version__
Out[2]: '2021.02.0+17.g7f8bb817'
```

---

Ok great! Yeah if we don't need it, let's skip it